tests/core/basic20/task.yaml

```diff
@@ -35,9 +35,11 @@ execute: |
     echo "Check that we booted with the rebuilt initramfs in the kernel snap"
     test -e /writable/system-data/the-tool-ran
 
-    echo "The recovery key is available"
-    test -e /var/lib/snapd/device/fde/recovery-key
-    echo "... and has the expected size"
-    test "$(stat --printf=%s /var/lib/snapd/device/fde/recovery-key)" = "16"
-    echo "... and has the expected owner and permissions"
-    test "$(stat --printf='%u:%g %a' /var/lib/snapd/device/fde/recovery-key)" = "0:0 600"
+    if [ -e /dev/tpm0 ]; then
```

Comment:
is this conditionality actually true atm?  we still alway encrypt atm?

Reply:
AIUI with this PR we will only encrypt if secbootCheckKeySealingSupported() returns true. This is true if we can talk to a TPM but not without. So this should be correct or am I missing something? AFAIK our 20.04 does have /dev/tpm0 so in practise this should not make any difference.

Reply:
thank you, no, it's me, I misread the code